apiserver/stateauthenticator/auth.go

```diff
@@ -235,14 +227,9 @@ func (a *Authenticator) AuthenticateLoginRequest(
 
 	// If you're a model worker api-caller using the model controller agent tag
 	// to ask questions about the non-controller model that you're running in.
-	if (isMachineTag || isControllerAgentTag) && !st.IsController() {
```

Comment:
@SimonRichardson This is the one bit I want to talk with you about.

Reply:
All ears :ear: